apps/web/src/components/workflow/preview/in-app/v2/Content.tsx

```diff
@@ -0,0 +1,219 @@
+import { Avatar } from '@mantine/core';
```

Comment:
```suggestion
```
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please avoid any direct references to mantine‏